enterprise/server/cmd/ci_runner/main.go

```diff
@@ -467,7 +467,7 @@ func run() error {
 
 	// Write setup logs to the current task's stderr (to make debugging easier),
 	// and also to the invocation.
-	ws.log = io.MultiWriter(os.Stderr, buildEventReporter)
+	ws.log = buildEventReporter
```

Comment:
Comment is slightly out of date. Either remove it or mention that build event reporter will handle all the logging, both to the invocation and to stderr.

Reply:
thanks - updated